airflow/www/templates/airflow/dags.html

```diff
@@ -233,11 +233,32 @@ <h2>DAGs</h2>
       const DAGS_INDEX = "{{ url_for('admin.index') }}";
       const ENTER_KEY_CODE = 13;
 
+      function createOrAppendToQueryString(parameter, value) {
+        var query_string_parameters = {};
+        if (typeof window.location.search !== "undefined") {
+          window.location.search.substr(1).split('&').forEach(function (query_string_parameter) {
+            query_string_parameter_delimeted = query_string_parameter.split('=');
```

Comment:
The `String.split` method supports the `limit` parameter, which should be used here.

Reference: 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/split

Reply:
Consider also splitting this function into smaller functions to increase its readability, for example, `queryStringParse`,` queryStringUnParse`,